examples/resources/bloxone_dhcp_server/resource.tf

```diff
@@ -26,3 +26,81 @@ resource "bloxone_dhcp_server" "example_with_options" {
     }
   ]
 }
+
+data "bloxone_dns_hosts" "example_dns_host" {
+  filters = {
+    name = "DNSHost01"
+  }
+}
+
+resource "bloxone_dns_view" "example_dns_view" {
+  name = "example_dns_view"
+
+  # Other Optional fields
+  comment = "An example view"
+  tags = {
+    site = "Site A"
+  }
+
+}
+
+resource "bloxone_dns_auth_zone" "example_auth_zone" {
+  fqdn         = "domain.com."
+  primary_type = "cloud"
+
+  # Other optional fields
```

Comment:
keep the auth_zone resource minimal here. Optional fields are not needed for auth_zone.

Reply:
done